packages/transactions/src/OutboundShipment/DetailView.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { FC } from 'react';
+import { useQueryClient } from 'react-query';
+import { useNavigate, useParams } from 'react-router';
+import { request, Transaction } from '@openmsupply-client/common';
+import { getMutation, getDetailQuery } from '../api';
+import { createDraftStore, useDraftDocument } from '../useDraftDocument';
+
+const queryFn = (id: string) => async (): Promise<Transaction> => {
+  const result = await request('http://localhost:4000', getDetailQuery(), {
```

Comment:
Need some common `http client` type thing to abstract this url and auth and stuff away later